Example/wifi_sta/firmware/src/app.c

```diff
@@ -680,35 +925,49 @@ void APP_Tasks ( void )
                         if(SYS_FS_FileWrite (ffsFileHandle, (void *)&currConfig, sizeof(currConfig)) == -1)
                         {
                             /* Failed to write to the file. */
-                            SYS_CONSOLE_MESSAGE("Failed to write the FFS configuration file\n");
+                            SYS_CONSOLE_MESSAGE("Failed to write the FFS configuration file\n\r");
                         }
                         else
-                            SYS_CONSOLE_MESSAGE("\n########################\nFFS configuration Saved!\n########################\n");                        
+                            SYS_CONSOLE_MESSAGE("\n\r########################\n\rFFS configuration Saved!\n\r########################\n\r");                        
                     }
                     SYS_FS_FileClose(ffsFileHandle);
                 }  
-                appData.state = APP_UNMOUNT_DISK;
+                appData.state = APP_STATE_SERVICE_TASKS;
             }   
             else
             {
-                SYS_CONSOLE_MESSAGE("\n########################\nFFS Failure!\n########################\n");                
+                SYS_CONSOLE_MESSAGE("\n\n\n\r###############################\n\rFFS Failure!\n\r###############################\n\r");                
             }
             WDT_Disable();
             SYS_TIME_TimerDestroy(ffsTmrHdl);
             LED_GREEN_Off();
-            appData.state = APP_UNMOUNT_DISK;
+            appData.state = APP_STATE_SERVICE_TASKS;
             break;    
         }
         
         case APP_STATE_SERVICE_TASKS:
         {
-
+            
+            break;
+        } 
+        case APP_ECC_Init:
+        {
+            extern ATCAIfaceCfg atecc608_0_init_data;
+            if (ATCA_SUCCESS == atcab_init(&atecc608_0_init_data))
```

Comment:
Here you can enable the ecc method or non-ecc method

Reply:
done